app/templates/components/forms/orders/order-form.hbs

```diff
@@ -279,13 +279,16 @@
         {{t 'Acknowledgement'}}
       </h4>
       <p>
-        {{t 'By clicking "{{orderState}}"' orderState=(if this.isPaidOrder (t "Proceed to Checkout") (t "Order Now"))}}
-        {{', I acknowledge that I have read and that I agree with the '}} 
-        <a href='/terms'>terms of services</a>{{t ' and ' }}<a href='/privacy'>privacy policy</a> {{t ' on this website.'}}      
+        <GetText
+          @message = {{t 'By clicking "{{orderState}}", I acknowledge that I have read and that I agree with the {{terms}} and {{privacy}} on this website.' 
+          orderState=(if this.isPaidOrder (t "Proceed to Checkout") (t "Order Now"))}} as |text placeholder|>
+          {{~#if (eq placeholder 'terms')}}<a href="/terms">terms of services</a>{{~/if~}}
+          {{~#if (eq placeholder 'privacy')}}<a href="/privacy">privacy policy</a>{{~/if~}}
```

Comment:
Why not use `terms=<a href="/terms">terms of services</a>` and `privacy=<a href="/privacy">privacy policy</a>` like in orderState?

Reply:
Was showing error

Reply:
![Screenshot from 2020-11-22 23-11-10](https://user-images.githubusercontent.com/55341933/99911076-0ae24c80-2d18-11eb-8688-5fa8e8b7402a.png)


This one

Reply:
You have to wrap that in string

Reply:
Tried and didn't worked with that too. Browser was showing no link only the whole text inside quotes